requirements/requirements-nlp.txt

```diff
@@ -1 +1 @@
-transformers<=2.4.0>=2.0.0
\ No newline at end of file
+transformers>=2.8.0
```

Comment:
why could not we use older versions of `transformers` ?

Reply:
`2.8.0` is... very new :)
why could not we work with `2.0.0` still?

Reply:
I need `DataCollatorForLanguageModeling` from transformers. It was published about 20 days ago. Maybe I can just add this class to utils with credentials to hf and use older version.

Reply:
I suppose, we could add `version` check
something like https://github.com/catalyst-team/catalyst/blob/master/catalyst/utils/seed.py#L25

Reply:
well, there is literally nothing like this before 2.8.0, so I can't do something like
```python
if parse(transformers.__version__) >= Version("2.8.0"):
    collate_fn = DataCollatorForLanguageModeling().collate_batch
else:
    # what should I do?)
    collate_fn = utils.DataCollatorForLanguageModeling().collate_batch   # ?
```
also I need `max_len` arg in PreTrainedTokenizer. I'm not sure about version, but this feature is pretty new too.